WordPress/Classes/Services/PostService.h

```diff
@@ -92,6 +92,21 @@ extern const NSUInteger PostServiceDefaultNumberToSync;
            success:(nullable void (^)(AbstractPost *post))success
            failure:(void (^)(NSError * _Nullable error))failure;
 
+/**
+ Saves a post to the server.
+ 
+ @param post The post or page to be saved
+ @param success A success block.  If the post object exists locally (in CoreData) when the upload
+ succeeds, then this block will also return a pointer to the updated local AbstractPost
+ object.  It's important to note this object may not be the same one as the `post` input
+ parameter, since if the input post was a revision, it will no longer exist once the upload
+ succeeds.
+ @param failure A failure block
+ */
+- (void)savePost:(AbstractPost *)post
```

Comment:
Same question about naming that I had in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/156